webapp/channels/src/selectors/drafts.ts

```diff
@@ -115,32 +115,80 @@ export function makeGetDraft() {
         rootId: '',
     });
 
-    return (state: GlobalState, channelId: string, rootId = '', storageKey = ''): PostDraft => {
-        let prefixStorageKey = StoragePrefixes.DRAFT;
-        let suffixStorageKey = channelId;
-        if (rootId) {
-            prefixStorageKey = StoragePrefixes.COMMENT_DRAFT;
-            suffixStorageKey = rootId;
-        }
-        const key = storageKey || `${prefixStorageKey}${suffixStorageKey}`;
-
-        const retrievedDraft = getGlobalItem<PostDraft>(state, key, DEFAULT_DRAFT);
-
-        // Check if the draft has the required values in its properties
-        const isDraftWithRequiredValues = typeof retrievedDraft.message !== 'undefined' && typeof retrievedDraft.uploadsInProgress !== 'undefined' && typeof retrievedDraft.fileInfos !== 'undefined';
-
-        // Check if draft's channelId or rootId mismatches with the passed one
-        const isDraftMismatched = retrievedDraft.channelId !== channelId || retrievedDraft.rootId !== rootId;
-
-        if (isDraftWithRequiredValues && !isDraftMismatched) {
-            return retrievedDraft;
-        }
-
-        return {
-            ...DEFAULT_DRAFT,
-            ...retrievedDraft,
-            channelId,
-            rootId,
-        };
-    };
+    // return createSelector(
```

Comment:
Seems you left some unneeded comments in this file.

Reply:
yup, removed now.